pandas/tests/io/json/test_pandas.py

```diff
@@ -738,11 +730,10 @@ def test_reconstruction_index(self):
         result = read_json(df.to_json())
         tm.assert_frame_equal(result, df)
 
-    def test_path(self):
+    def test_path(self, float_frame):
         with tm.ensure_clean("test.json") as path:
             for df in [
-                self.frame,
-                self.frame2,
+                float_frame,
```

Comment:
Similar to above comment, self.frame and self.frame2 are actually different in that the former uses default column labels whereas the latter used ABCD; I didn't think that distinction was worth creating a separate fixture or object within this test

Reply:
sure, but this is a historical accident, having a smaller set of common fixtures is better